cls/SourceControl/Git/Modification.cls

```diff
@@ -0,0 +1,14 @@
+/// Class to store information about each file that is modified by git pull
+Class SourceControl.Git.Modification Extends %RegisteredObject
+{
+
+/// path of the file
+Property externalName As %String;
+
+/// Name in IRIS SourceControl.Git.Modification
+Property internalName As %String;
+
+/// Type of change (A|C|D|M|R|T|U|X|B). See git diff documentation.
+Property changeType As %String;
```

Comment:
Longer-term, would be nice to have more of an API around this than using the single git character code. That'll be a follow-up.